backend/Models/Database/ConservationStatuses.cs

```diff
@@ -0,0 +1,9 @@
+namespace WhaleSpotting.Models.Database
+{
+    public class ConservationStatuses
+    {
+        public int Id { get; set; }
+        public char Code { get; set; }
```

Comment:
This won't be enough to hold the code, as a `char` can only represent a single letter, but the codes are all 2 letters. This will have to be a `string`